communities/microgalaxy/lab/sections/2_microbial_isolates.yml

```diff
@@ -268,3 +712,21 @@ tabs:
       https://training.galaxyproject.org//topics/variant-analysis/tutorials/microbial-variants/tutorial.html
     button_tip: View tutorial
     button_icon: tutorial
+
+- id: learning-pathway
+  title: Learning pathways
```

Comment:
Can you add https://training.galaxyproject.org/training-material/learning-pathways/genome-annotation-prokaryote.html?